src/common/common-macros-private.h

```diff
@@ -1,3 +1,18 @@
+/*
+ * Copyright 2009-present MongoDB Inc.
```

Comment:
```suggestion
 * Copyright 2009-present MongoDB, Inc.
```
Add missing comma (significant). Also applies to other files in this PR.